### PR TITLE
snippets styling for new docs

### PR DESCRIPTION
**Pull request by jwunderl**
re: https://github.com/microsoft/pxt/issues/5987

move buttons above snippet and subheaders to the right

updated:

also minor patch to headers; a few pages have some weird headers with no content in them just to add some extra targets (e.g. https://github.com/microsoft/pxt/blob/master/common-docs/blocks/on-start.md), which can cause a slightly annoying double line; the h2 changes in docs.less just makes it so those get collapsed when there is more than a single consecutive h2s with no content in between them, and that the h2s otherwise always get a border unless they are the very first thing on the page. Can separate that out if preferred

### CODE REVIEW

**Comment by abchatra** [comment]
Test with python enabled as well

**Comment by jwunderl** [comment]
here's with python enabled on https://arcade.makecode.com/reference/game/over

File: pxtrunner/renderer.ts [hunk 1]
```diff
@@ -55,13 +51,17 @@ namespace pxt.runner {
         }
     }
 
+    function appendBlocks($parent: JQuery, $svg: JQuery) {
+        $parent.append($(`<div class="ui content blocks"/>`).append($svg));
+    }
+
     function appendJs($parent: JQuery, $js: JQuery, woptions: WidgetOptions) {
-        $parent.append($('<div class="ui content js"><div><i class="ui icon xicon js"/>JavaScript</div></div>').append($js));
+        $parent.append($(`<div class="ui content js"><div class="subheading"><i class="ui icon xicon js"/>${lf("JavaScript")}</div></div>`).append($js));
```
Comment (pelikhan) [inline pxtrunner/renderer.ts]: we are not localizing JavaScript in the app.

File: pxtrunner/renderer.ts [hunk 2]
```diff
@@ -55,13 +51,17 @@ namespace pxt.runner {
...
         highlight($js);
     }
 
     function appendPy($parent: JQuery, $py: JQuery, woptions: WidgetOptions) {
-        $parent.append($('<div class="ui content py"><div><i class="ui icon xicon python"/>Python</div></div>').append($py));
+        $parent.append($(`<div class="ui content py"><div class="subheading"><i class="ui icon xicon python"/>${lf("Python")}</div></div>`).append($py));
```
Comment (pelikhan) [inline pxtrunner/renderer.ts]: No localizing Python in the app

**Comment by jwunderl** [comment]
Ahh that makes sense, I thought I saw a comment to localize it in the much reverted PR but I might have read it backwards

**Pull request merged**